scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -12,7 +12,12 @@ try {
             node(NODE_LABEL) {
                 def insideDocker = null
 
-                env.BUILD_NUMBER_DIR = currentBuild.number
+                if (params.VERSION.isEmpty()) {
+                    env.BUILD_NUMBER_DIR = currentBuild.number
```

Comment:
It would be great to store the latest version in `gradle.properties` and another stage to pipeline that will increase the number if the proceeding were successful.